.gitignore

```diff
@@ -19,6 +19,7 @@ proptest-regressions/
 # Devenv
 .devenv*
 devenv.local.nix
+.direnv
```

Comment:
advise putting local env gitignores like this in your system global git ignore config